### PR TITLE
Source Slack: add suggested streams

### PR DESCRIPTION
**Pull request by arsenlosenko**
## What
Add list of suggested streams for Source Slack

## How
Update `metadata.yaml` with list of streams in order of their popularity

### CODE REVIEW

**Review by davydov-d — APPROVED** [review]

**Review by lazebnyi — APPROVED** [review]

**Comment by evantahler** [comment]
Looks like you've got some merge conflicts here... Please clean up the PR, and we can get this merged in!

**Comment by arsenlosenko** [comment]
@evantahler diff with master is a bit messy, looks like I fixed the conflicts, if issue remains here I'll just create a new PR, changes here are minor anyway

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql test report (commit [`ac2ac1312c`](https://github.com/airbytehq/airbyte/pull/30580/commits/ac2ac1312c6157e6b1b0e494312fb33149402435)) - ✅

⏲️  Total pipeline duration: 27mn38s 

| Step                                                                | Result   |
|:--------------------------------------------------------------------|:---------|
| Build connector tar                                                 | ✅        |
| Build source-mysql docker image for platform linux/x86_64           | ✅        |
| Java Connector Unit Tests                                           | ✅        |
| Java Connector Integration Tests                                    | ✅        |
| Acceptance tests                                                    | ✅        |
| Validate airbyte-integrations/connectors/source-mysql/metadata.yaml | ✅        |
| Connector version semver check                                      | ✅        |
| QA checks                                                           | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/arsenlosenko_source-slack-add-suggested-streams/1695745435/ac2ac1312c6157e6b1b0e494312fb33149402435/source-mysql/3.1.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:ac2ac1312c6157e6b1b0e494312fb33149402435)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-mysql-strict-encrypt test report (commit [`ac2ac1312c`](https://github.com/airbytehq/airbyte/pull/30580/commits/ac2ac1312c6157e6b1b0e494312fb33149402435)) - ✅

⏲️  Total pipeline duration: 07mn56s 

| Step                                                                               | Result   |
|:-----------------------------------------------------------------------------------|:---------|
| Build connector tar                                                                | ✅        |
| Build source-mysql-strict-encrypt docker image for platform linux/x86_64           | ✅        |
| Java Connector Unit Tests                                                          | ✅        |
| Java Connector Integration Tests                                                   | ✅        |
| Acceptance tests                                                                   | ✅        |
| Validate airbyte-integrations/connectors/source-mysql-strict-encrypt/metadata.yaml | ✅        |
| Connector version semver check                                                     | ✅        |
| QA checks                                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/arsenlosenko_source-slack-add-suggested-streams/1695745435/ac2ac1312c6157e6b1b0e494312fb33149402435/source-mysql-strict-encrypt/3.1.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:ac2ac1312c6157e6b1b0e494312fb33149402435)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql-strict-encrypt test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-slack test report (commit [`ac2ac1312c`](https://github.com/airbytehq/airbyte/pull/30580/commits/ac2ac1312c6157e6b1b0e494312fb33149402435)) - ❌

⏲️  Total pipeline duration: 58mn31s 

| Step                                                                | Result   |
|:--------------------------------------------------------------------|:---------|
| Connector package install                                           | ✅        |
| Build source-slack docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                          | ✅        |
| Acceptance tests                                                    | ❌        |
| Code format checks                                                  | ✅        |
| Validate airbyte-integrations/connectors/source-slack/metadata.yaml | ✅        |
| Connector version semver check                                      | ✅        |
| QA checks                                                           | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/arsenlosenko_source-slack-add-suggested-streams/1695745435/ac2ac1312c6157e6b1b0e494312fb33149402435/source-slack/0.3.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:ac2ac1312c6157e6b1b0e494312fb33149402435)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-slack test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-slack test report (commit [`3fbebf3c86`](https://github.com/airbytehq/airbyte/pull/30580/commits/3fbebf3c86b7e805480a921f38d50e42798db8ee)) - ❌

⏲️  Total pipeline duration: 27mn07s 

| Step                                                                | Result   |
|:--------------------------------------------------------------------|:---------|
| Connector package install                                           | ✅        |
| Build source-slack docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                          | ✅        |
| Acceptance tests                                                    | ❌        |
| Code format checks                                                  | ✅        |
| Validate airbyte-integrations/connectors/source-slack/metadata.yaml | ✅        |
| Connector version semver check                                      | ✅        |
| QA checks                                                           | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/arsenlosenko_source-slack-add-suggested-streams/1695758332/3fbebf3c86b7e805480a921f38d50e42798db8ee/source-slack/0.3.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:3fbebf3c86b7e805480a921f38d50e42798db8ee)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-slack test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-slack test report (commit [`9df13cf377`](https://github.com/airbytehq/airbyte/pull/30580/commits/9df13cf377dcc6068ea28b1c05a10135a6cf03df)) - ✅

⏲️  Total pipeline duration: 11mn43s 

| Step                                                                | Result   |
|:--------------------------------------------------------------------|:---------|
| Connector package install                                           | ✅        |
| Build source-slack docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                          | ✅        |
| Acceptance tests                                                    | ✅        |
| Code format checks                                                  | ✅        |
| Validate airbyte-integrations/connectors/source-slack/metadata.yaml | ✅        |
| Connector version semver check                                      | ✅        |
| QA checks                                                           | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/arsenlosenko_source-slack-add-suggested-streams/1695801035/9df13cf377dcc6068ea28b1c05a10135a6cf03df/source-slack/0.3.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:9df13cf377dcc6068ea28b1c05a10135a6cf03df)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-slack test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-slack test report (commit [`1ea2554e43`](https://github.com/airbytehq/airbyte/pull/30580/commits/1ea2554e43fa7f6d25f612b13be2b60dd526d241)) - ✅

⏲️  Total pipeline duration: 11mn56s 

| Step                                                                | Result   |
|:--------------------------------------------------------------------|:---------|
| Connector package install                                           | ✅        |
| Build source-slack docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                          | ✅        |
| Acceptance tests                                                    | ✅        |
| Code format checks                                                  | ✅        |
| Validate airbyte-integrations/connectors/source-slack/metadata.yaml | ✅        |
| Connector version semver check                                      | ✅        |
| QA checks                                                           | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/arsenlosenko_source-slack-add-suggested-streams/1695802955/1ea2554e43fa7f6d25f612b13be2b60dd526d241/source-slack/0.3.2/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:1ea2554e43fa7f6d25f612b13be2b60dd526d241)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-slack test
```

File: docs/integrations/sources/slack.md [hunk 1]
```diff
@@ -76,10 +40,10 @@ We recommend creating a restricted, read-only key specifically for Airbyte acces
 2. In the left navigation bar, click **Sources**. In the top-right corner, click **+new source**.
 3. On the Set up the source page, enter the name for the Slack connector and select **Slack** from the Source type dropdown.
 4. Select `Authenticate your account` and log in and Authorize to the Slack account.
-5. Enter your `start_date`.
-6. Enter your `lookback_window`.
-7. Enter your `join_channels`.
-8. Enter your `channel_filter`.
+5. **Required** Enter your `start_date`.
```
[the remark this answers is not in the document]
Comment (arsenlosenko) [reply docs/integrations/sources/slack.md:43]: okay, now I finally see, `slack.md` is for Airbyte docs and `slack.inapp.md` is for Cloud UI, sorry I missed that, updated this exact doc you have mentioned

**Comment by airbyte-oss-build-runner** [comment]
##  source-slack test report (commit [`fda0c1ef33`](https://github.com/airbytehq/airbyte/pull/30580/commits/fda0c1ef33e0a12f02301e6f9cdbedf6e6a5e17b)) - ❌

⏲️  Total pipeline duration: 71mn07s 

| Step                                                      | Result   |
|:----------------------------------------------------------|:---------|
| Connector package install                                 | ✅        |
| Build source-slack docker image for platform linux/x86_64 | ✅        |
| Unit tests                                                | ✅        |
| Acceptance tests                                          | ❌        |
| Code format checks                                        | ✅        |
| Validate metadata for source-slack                        | ✅        |
| Connector version semver check                            | ✅        |
| Connector version increment check                         | ✅        |
| QA checks                                                 | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/arsenlosenko_source-slack-add-suggested-streams/1695911504/fda0c1ef33e0a12f02301e6f9cdbedf6e6a5e17b/source-slack/0.3.3/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:fda0c1ef33e0a12f02301e6f9cdbedf6e6a5e17b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-slack test
```

**Comment by airbyte-oss-build-runner** [comment]
##  source-slack test report (commit [`7c28f4bc51`](https://github.com/airbytehq/airbyte/pull/30580/commits/7c28f4bc51f4d64d74c61cb78d8b4665a5553cf9)) - ❌

⏲️  Total pipeline duration: 126mn11s 

| Step                                                      | Result   |
|:----------------------------------------------------------|:---------|
| Connector package install                                 | ✅        |
| Build source-slack docker image for platform linux/x86_64 | ✅        |
| Unit tests                                                | ✅        |
| Acceptance tests                                          | ❌        |
| Code format checks                                        | ✅        |
| Validate metadata for source-slack                        | ✅        |
| Connector version semver check                            | ✅        |
| Connector version increment check                         | ✅        |
| QA checks                                                 | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/arsenlosenko_source-slack-add-suggested-streams/1695916101/7c28f4bc51f4d64d74c61cb78d8b4665a5553cf9/source-slack/0.3.3/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:7c28f4bc51f4d64d74c61cb78d8b4665a5553cf9)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-slack test
```

**Pull request merged**